### ISSUE TITLE
Cannot enable three-finger drag via Marco (accessibility feature)

### ISSUE THREAD

**Issue by foxwhat** [mate-desktop/marco#805]
Libinput 1.28 now supports three finger drag (i.e an accessibility feature that uses dragging three fingers as an equivalent to click and drag) but I cannot find a way to turn it on. 

I have been told I should enable it via compositor settings but there is no setting in the compositor I can find. I assume it needs to be added and doesn't exist yet because it's a relatively new feature. 

Please can you add a way to enable this feature. 

https://gitlab.freedesktop.org/libinput/libinput/-/merge_requests/1042

(Debian 13, Mate, Marco, x11)
